src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementUserBadge.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Badge,
+} from '@edx/paragon';
+
+import { USER_BADGE_MAP } from '../../data/constants';
+
+const LicenseManagementUserBadge = ({ userStatus }) => {
+  const badgeLabel = USER_BADGE_MAP[userStatus]
+    ? USER_BADGE_MAP[userStatus]
+    : USER_BADGE_MAP.UNDEFINED;
```

Comment:
I think I commented this above, but if there isn't a valid `userStatus`, let's not render a badge at all (empty cell or "-") so it's not misleading.